src/index.js

```diff
@@ -25,6 +25,7 @@ export type IEdgeType = IEdge;
 export { default as GraphUtils } from './utilities/graph-util';
 export { default as Node } from './components/node';
 export type INodeType = INode;
+// eslint-disable-next-line prettier/prettier
```

Comment:
Fixes an annoying problem with prettier trying to fix the line breaks when it doesn't need to.